tests/e2e/ansible/install_docker.yml

```diff
@@ -0,0 +1,69 @@
+# Copyright 2022 Red Hat
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Install docker on the host as it is used to build the operator.
+#
+---
+# It assume that if docker is installed then it is proper configure so it
+# doesn't do anything else. This is likely true when running on baremetal
+# machines.
+- name: Check docker is installed
+  shell: command -v docker >/dev/null 2>&1
+  register: docker_exist
+  ignore_errors: yes
+- name: Handle docker installation on Ubuntu
+  block:
+    - name: Install docker dependencies
+      package:
+        name:
+          - ca-certificates
+          - curl
+          - gnupg
+          - lsb-release
+        state: present
+    - name: Add docker repo GPG key
+      apt_key:
+        url: https://download.docker.com/linux/ubuntu/gpg
+        keyring: /etc/apt/trusted.gpg.d/docker.gpg
+        state: present
+    - name: Add docker repo
+      apt_repository:
+        repo: "deb [arch=amd64 signed-by=/etc/apt/trusted.gpg.d/docker.gpg] https://download.docker.com/linux/ubuntu {{ ansible_lsb.codename }} stable"
+        filename: docker
+        update_cache: yes
+        state: present
+    - name: Install docker packages
+      package:
+        name:
+          - docker-ce
+          - docker-ce-cli
```

Comment:
`containerd.io` need to be installed ? 
Ref: https://docs.docker.com/engine/install/ubuntu/

Reply:
According to the CentOS guide, yes: https://docs.docker.com/engine/install/centos/

Reply:
yes, it does. I didn't notice that because in `tests/e2e/ansible/install_containerd.yml` I end up installing containerd. But it makes sense to get it installed in install_docker.yml as I am trying to make those playbook self-contained.

Good catch. Gonna fix it.